src/user_lists/serializers.py

```diff
@@ -18,3 +24,115 @@ class Meta:
+            "slug",
+            "authors",
+            "abstract",
+            "doi",
+            "image_url",
+            "renderable_text",
+            "document_type",
+            "score",
+            "discussion_count",
+            "hubs",
+            "created_by",
+            "unified_document",
+            "unified_document_id",
+            "fundraise",
+            "grant",
+        ]
+    },
+    "doc_duds_get_hubs": {"_include_fields": ["id", "name", "slug"]},
+    "doc_duds_get_created_by": {"_include_fields": ["id", "author_profile", "first_name", "last_name"]},
+    "doc_duds_get_fundraise": {
+        "_include_fields": [
+            "id",
+            "status",
+            "goal_amount",
+            "goal_currency",
+            "start_date",
+            "end_date",
+            "amount_raised",
+            "contributors",
+            "created_by",
+        ]
+    },
+    "doc_duds_get_grant": {
+        "_include_fields": [
+            "id",
+            "status",
+            "amount",
+            "currency",
+            "organization",
+            "description",
+            "start_date",
+            "end_date",
+            "created_by",
+            "contacts",
+            "applications",
+        ]
+    },
+    "pap_dps_get_authors": {"_include_fields": ["id", "first_name", "last_name", "author_profile"]},
+    "pap_dps_get_unified_document": {"_include_fields": ["id", "fundraise", "grant"]},
+    "doc_dps_get_authors": {"_include_fields": ["id", "first_name", "last_name", "author_profile"]},
+    "doc_dps_get_unified_document": {"_include_fields": ["id", "fundraise", "grant"]},
+    "pch_dfs_get_contributors": {"_include_fields": ["id", "author_profile", "first_name", "last_name", "profile_image"]},
+    "pch_dfs_get_created_by": {"_include_fields": ["id", "first_name", "last_name", "profile_image", "author_profile"]},
+    "pch_dgs_get_created_by": {"_include_fields": ["id", "first_name", "last_name", "profile_image", "author_profile"]},
+    "pch_dgs_get_contacts": {"_include_fields": ["id", "first_name", "last_name", "profile_image", "author_profile"]},
+}
+
+
+class ListItemSerializer(DefaultAuthenticatedSerializer):
+    unified_document = serializers.PrimaryKeyRelatedField(
+        queryset=ResearchhubUnifiedDocument.objects.filter(is_removed=False), required=True
+    )
+
+    class Meta:
+        model = ListItem
+        fields = ["id", "parent_list", "unified_document", "created_date", "created_by"]
+        read_only_fields = ["id", "created_date", "created_by"]
+
+
+class ListItemDetailSerializer(ListItemSerializer):
+    unified_document_data = serializers.SerializerMethodField()
+
+    class Meta(ListItemSerializer.Meta):
+        fields = ListItemSerializer.Meta.fields + ["unified_document_data"]
+
+    def get_unified_document_data(self, obj):
+        context = {**self.context, **_UNIFIED_DOC_CONTEXT}
+        try:
+            return DynamicUnifiedDocumentSerializer(
```

Comment:
We are no longer using dynamic serializers with contexts, etc... as they lead to a ton of issues and are hard to debug. Just add all the methods/fields you need into your own serializer and that's it!